File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/StaffSmall.java
package com.example.owner_pc.animechecker.model.entity;

import java.io.Serializable;

/**
 * Created by owner-PC on 2017/05/20.
 */

public class StaffSmall implements Serializable {
    public int id;
    public String nameFirst;
    public String nameLast;
    public String language;
    public String imageUrlLge;
    public String imageUrlMed;
    public int linkId;
    public String role;

    @Override
    public String toString() {
        return "StaffSmall{" +
                "id=" + id +
                ", nameFirst='" + nameFirst + '\'' +
                ", nameLast='" + nameLast + '\'' +
                ", language='" + language + '\'' +
                ", imageUrlLge='" + imageUrlLge + '\'' +
                ", imageUrlMed='" + imageUrlMed + '\'' +
                ", linkId=" + linkId +
                ", role='" + role + '\'' +
                '}';
    }
//
    public StaffSmall() {
        this.id = -1;
        this.nameFirst = "";
        this.nameLast = "";
        this.language = "";
        this.imageUrlLge = "";
        this.imageUrlMed = "";
        this.linkId = -1;
        this.role = "";
    }
}

File: app/src/main/java/com/example/owner_pc/animechecker/model/entity/StaffSmall.kt
package com.example.owner_pc.animechecker.model.entity

import java.io.Serializable

/**
 * Created by owner-PC on 2017/05/20.
 */

class StaffSmall : Serializable {
    var id: Int = 0
    var nameFirst: String
    var nameLast: String
    var language: String
    var imageUrlLge: String
    var imageUrlMed: String
    var linkId: Int = 0
    var role: String

    //
    init {
        this.id = -1
        this.nameFirst = ""
        this.nameLast = ""
        this.language = ""
        this.imageUrlLge = ""
        this.imageUrlMed = ""
        this.linkId = -1
        this.role = ""
    }

    override fun toString(): String {
        return "StaffSmall{" +
                "id=" + id +
                ", nameFirst='" + nameFirst + '\'' +
                ", nameLast='" + nameLast + '\'' +
                ", language='" + language + '\'' +
                ", imageUrlLge='" + imageUrlLge + '\'' +
                ", imageUrlMed='" + imageUrlMed + '\'' +
                ", linkId=" + linkId +
                ", role='" + role + '\'' +
                '}'
    }
}
